ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/SupportsPeras.hs

```diff
@@ -43,6 +44,7 @@ import Quiet (Quiet (..))
 
 newtype PerasRoundNo = PerasRoundNo {unPerasRoundNo :: Word64}
   deriving Show via Quiet PerasRoundNo
+  deriving Semigroup via Sum Word64
```

Comment:
Hmm, for comparison, [`SlotNo`](https://base.cardano.intersectmbo.org/cardano-slotting/Cardano-Slotting-Slot.html#t:SlotNo) doesn't provide a particular `Semigroup`, but rather gives `Num`. `Num` of course isn't ideal, but unless you are opposed, I think we could follow suit.

(Also, *if* we add `Semigroup`, we should also add `Monoid` I think.)